### DISCUSSION TITLE
Use with IPFS

### DISCUSSION THREAD

**Discussion by joewagner** [nuxt/nuxt#17646] [Questions]
I'm excited to see SSG land.  I was hoping to use the ability to generate a static site to use Nuxt3 with Web3 apps via IPFS.  This requires that the generated site uses relative paths.  
Does anyone know how to accomplish this?  I looked through the nxut3 docs and nitro docs but I can't find any solutions.
Thanks for the help!

**Comment by xlanex6** [comment]
Hi @joewagner 

Can you bring some exemple of your problem.

do you play with ipfs js SDK ? as plugin ? as module

**Reply by joewagner** [reply]
Hey @xlanex6 
Sorry for the vauge issue.  The problem I'm having getting this deployed on IPFS is that when the static site is generated all of the paths start with `/`.   In order for a site to work with IPFS the path cannot start with a slash.
For a specific example, when I run `nuxi generate` a file named `index.html` is created and in that file there is a script tag with an attribute  `src="/_nuxt/entry-895c5627.mjs"`.  To work with IPFS it would need it to be `src="_nuxt/entry-895c5627.mjs"`.

Steps to reproduce
 1. generate a site
 2. open the index.html file via your file system.

Thanks for the help!

**Reply by xlanex6** [reply]
have you try https://github.com/Developerayo/deploy-nuxtjs-using-ipfs-on-fleek ?

**Reply by joewagner** [reply]
I hadn't seen that yet, look promising!

**Reply by joewagner** [reply]
That lib is specific to Nuxt 2.  I have been able to get Nuxt 2 working with IPFS or the file system.  I haven't found anything like this that works with Nuxt 3 yet, but I'll keep looking.

**Reply by xlanex6** [reply]
If you 're using nuxt 3 , fulls static is not fully support

https://v3.nuxtjs.org/community/roadmap/#%EF%B8%8F-roadmap

**Reply by joewagner** [reply]
That makes sense.  I saw that #3161 was closed and was hoping a static site served from a file system would work.

**Reply by xlanex6** [reply]
Closed but migrate to nitro .... 
I hope also, it will be available soon
